qml/CredentialItem.qml

```diff
@@ -19,7 +19,7 @@ Rectangle {
                                                    && credential.period !== 30)
                                                   || (credential.touch
                                                       && !isExpired)))
-    readonly property bool isExpired: timeLeft < 0
+    readonly property bool isExpired: timeLeft <= 0
```

Comment:
I think this change should be reverted to avoid the code starting out in a greyed out state.

Reply:
Sure 👍